tests/unit/time_units/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+common:
+  tags: time_units
+  type: unit
+tests:
+  utilities.time_units.z_tmcvt:
+    extra_configs:
+      # Arbitrary property so that test element is not empty
+      - CONFIG_ZTEST=y
```

Comment:
Not critical or blocking, but this is weird yaml. Does `utilities.time_units.z_tmcvt: {}` work?

Reply:
I'm a YAML n00b, so my guess is it will. Are there other instances of that in-tree?

Reply:
Fixed!

Reply:
> I'm a YAML n00b, so my guess is it will.

I guess you tested it? My working tree is  busy with another branch rn, which is why I phrased it as a question

> Are there other instances of that in-tree?

I don't know -- did you copy/paste it from somewhere?

My example is definitely valid YAML, it just creates an empty dictionary in the test specification:

```
In [21]: yaml_str = '''
    ...: common:
    ...:   tags: time_units
    ...:   type: unit
    ...: tests:
    ...:   utilities.time_units.z_tmvct: {}
    ...: '''

In [22]: yaml.safe_load(yaml_str)
Out[22]: 
{'common': {'tags': 'time_units', 'type': 'unit'},
 'tests': {'utilities.time_units.z_tmvct': {}}}
```

from the comment on line 7 above, it looked to me like someone was looking for a way to have a dict key which mapped to an empty dict, but didn't know how to spell that in YAML.

Reply:
> > I'm a YAML n00b, so my guess is it will.
> I guess you tested it?

Yep

> > Are there other instances of that in-tree?
> I don't know -- did you copy/paste it from somewhere?

There is 1 instance in-tree that I found with `grep` and I only knew of it after you mentioned. But no, I did not copy-paste, but knew from my limited yaml experience that empty yaml attributes do not parse properlly.

Reply:
> but knew from my limited yaml experience that empty yaml attributes do not parse properlly.

Gotcha. The secret trick is that YAML is actually a superset of JSON, so if you don't know how to do something in YAML  but you do in JSON, it works as you'd expect.

As an extreme example:

```
In [2]: yaml_str = '''{'common': {'tags': 'time_units', 'type': 'unit'},
   ...:  'tests': {'utilities.time_units.z_tmvct': {}}}'''

In [3]: yaml.safe_load(yaml_str)
Out[3]: 
{'common': {'tags': 'time_units', 'type': 'unit'},
 'tests': {'utilities.time_units.z_tmvct': {}}}
```